### PR TITLE
Loaded the content page only once

### PR DESCRIPTION
**Pull request by shahanemahesh**

### CODE REVIEW

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Ginger/Ginger/RunSetPageLib/RunsetOperationsPage.xaml.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6392702&bid=20732579)

File: Ginger/Ginger/RunSetPageLib/RunsetOperationsPage.xaml.cs [hunk 1]
```diff
@@ -90,12 +90,29 @@ public void Init(RunSetConfig runSetConfig)
...
 
+        RunSetActionEditPage runSetActionEditPage;
         private void RunSetActionsGrid_RowChangedEvent(object sender, EventArgs e)
         {
             RunSetActionEditFrame.Content = null;
             if (RunSetActionsGrid.CurrentItem != null)
             {
-                RunSetActionEditPage RSAEP = new RunSetActionEditPage((RunSetActionBase)RunSetActionsGrid.CurrentItem);
+                RunSetActionEditPage RSAEP = null;
+                if (((Amdocs.Ginger.Repository.RepositoryItemBase)RunSetActionsGrid.CurrentItem).ItemName == "Open ALM Defects")
```
Comment (JinendraGhodke) [inline Ginger/Ginger/RunSetPageLib/RunsetOperationsPage.xaml.cs:100]: Why specific handling needed for Defects Operation. We have other runset operations like Send email and report etc. 
So either this issue exist also for those pages. Or we need some specific fix for Defects Operation in that page.

**Review by JinendraGhodke — CHANGES_REQUESTED** [review]
Added comment

File: Ginger/Ginger/RunSetPageLib/RunsetOperationsPage.xaml.cs (hunk 1 above)
Comment (shahanemahesh) [reply Ginger/Ginger/RunSetPageLib/RunsetOperationsPage.xaml.cs:100]: Here what is happening that every time the operation is selected from the grid then it again instantiating the component page and every time the newly created page instance is displayed, now for rest of operations it is also behaving the same way but i.e. again creating the instance of selected operations page there it is not messing the data already selected but here in this case "GingerCore.GeneralLib.BindingHandler.ObjFieldBinding(RadioDefectsOpeningModeForAll, RadioButton.IsCheckedProperty, runSetActionAutomatedALMDefects, nameof(RunSetActionAutomatedALMDefects.DefectsOpeningModeForAll));" the binding used for the radio button is not working correctly when the page is loaded 2nd and after that. when this binding is called for the radio button it resetting all selections for that control.

Also why to recreate the page again and again when we have its instance then it should be used to load it?

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Ginger/Ginger/RunSetPageLib/RunsetOperationsPage.xaml.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6392702&bid=20732579)

**Review by JinendraGhodke — APPROVED** [review]

**Pull request merged**